sqrl-openai/src/main/java/com/datasqrl/openai/util/P99LatencyTracker.java

```diff
@@ -0,0 +1,35 @@
+package com.datasqrl.openai.util;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class P99LatencyTracker {
```

Comment:
I’m not completely satisfied that I had to create this class manually. It would have been much better to use the existing Histogram class here. Since this is just a temporary solution until we can integrate the org.apache.flink:flink-metrics-dropwizard dependency, I opted not to implement a more performant solution for now.

Reply:
I see your point. I think this is good enough for now and we can adjust once we get customer feedback.